src/folder/schema.ts

```
@@ -0,0 +1,147 @@
+  socialLinks: z.record(z.string().url()).optional().nullable(),
+});
+
+
+export const projectSchema = z.object({
+  title: z.string().min(1).max(255),
+  subtitle: z.string().max(255).optional().nullable(),
+  description: z.string().min(1),
+  goal: monetaryString(), // Using custom transformer
+  minimumPledge: monetaryString(), // Using custom transformer
+  currency: z.enum(currencyEnum.enumValues).optional(),
+  endDate: z.date().refine((date) => date > new Date(), { message: "End date must be in the future" }),
```

ESLint

[error] Unexpected parentheses around single function argument having a body with no curly braces.

(style/arrow-parens)